modules/server/src/middleware/buildAggregations/createFieldAggregation.js

```diff
@@ -7,18 +7,26 @@ import { STATS, HISTOGRAM, BUCKETS, BUCKET_COUNT, CARDINALITY, TOPHITS } from '#
 const MAX_AGGREGATION_SIZE = 300000;
 const HISTOGRAM_INTERVAL_DEFAULT = 1000;
 const CARDINALITY_DEFAULT_PRECISION_THRESHOLD = 40000; // max precision for ES6-7
+const RANGES_DEFAULT = [{ from: 0 }];
 
 const createNumericAggregation = ({ type, field, graphqlField }) => {
 	const args = get(graphqlField, [type, '__arguments', 0]) || {};
+	const options =
+		type === HISTOGRAM
+			? {
+					interval: get(args, 'interval.value') || HISTOGRAM_INTERVAL_DEFAULT,
+				}
+			: type === RANGE
+				? {
+						ranges: get(args, 'ranges.value') || RANGES_DEFAULT,
+					}
+				: {};
+
 	return {
 		[`${field}:${type}`]: {
 			[type]: {
 				field,
-				...(type === HISTOGRAM
-					? {
-							interval: get(args, 'interval.value') || HISTOGRAM_INTERVAL_DEFAULT,
-						}
-					: {}),
+				...options,
```

Comment:
adds `ranges` to numeric aggregation gql query, allowing arg of ranges to be passed

Reply:
are there tests added for this support?